packages/webpack-preset-accurapp/package.json

```diff
@@ -5,11 +5,11 @@
   "repository": "accurat/accurapp",
   "license": "MIT",
   "engines": {
-    "node": ">=6.5"
+    "node": ">=7"
   },
-  "keywords": [
```

Comment:
I think we can keep the keywords list, maybe adding some more interesting keyword like
`preset`,`webpack`,`accurat`

Reply:
ci sta, fatto